gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -481,6 +481,7 @@ def run_generate_trio_stats(
     rmt = rmt.filter_cols(hl.is_defined(vmt.cols()[rmt.col_key]))
 
     mt = hl.vds.to_dense_mt(hl.vds.VariantDataset(rmt, vmt))
+    mt = mt.checkpoint(hl.utils.new_temp_file("trio_stats_dense", "mt"), overwrite=True)
```

Comment:
```suggestion
    mt = mt.checkpoint(hl.utils.new_temp_file("trio_stats_dense", "mt"))
```

Reply:
the overwrite arg isn't needed when using new_temp_file

Reply:
L560, L621, L664 had overwrite too, should I remove them?

Reply:
yeah those can be removed too